tests/test_validator_integration.py

```
@@ -0,0 +1,975 @@
+
+import ast
+import inspect
+import os
+from pathlib import Path
+from typing import Any
+
+import pytest
+import yaml
+
+# ---------------------------------------------------------------------------
+# Lightweight helpers – no heavy DL imports at module scope
```

Ruff (0.15.1)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)